packages/js/src/workouts/tailwind-components/Stepper.js

```diff
@@ -171,11 +174,13 @@ TailwindStep.propTypes = {
 	stepIndex: PropTypes.number.isRequired,
 	isLastStep: PropTypes.bool.isRequired,
 	setActiveStepIndex: PropTypes.func.isRequired,
-	saveStep: PropTypes.func,
+	beforeContinue: PropTypes.func,
 	activeStepIndex: PropTypes.number.isRequired,
 };
 TailwindStep.defaultProps = {
-	saveStep: () => { },
+	beforeContinue: () => {
+		return true;
+	},
```

Comment:
I know it's a bit of type juggling but I think you can set `null` as a defaultprop for not required functions, which makes it easier to check their presence.

Though this solution is probably also fine, let's keep it for now (or _forever_).